src/components/flash/__spec__.js

```diff
@@ -215,12 +215,14 @@ describe('Flash', () => {
 
     describe('if the text does contains ::more::', () => {
       it('returns the text and creates an alert', () => {
-        spyOn(defaultInstance, 'toggleDialog');
+        const toggleDialogSpy = jasmine.createSpy();
```

Comment:
Yeah sure, this is all related to removing the `bind` that was required whenever `toggleDialog` was called previously to pass in an argument.

`toggleDialog` has now been refactored to take the title as an argument (defined using the fat arrow so that `this` is automatically bound) and it returns the function that `onClick` handler can use.